packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/gettingatoken.md

```diff
@@ -0,0 +1,24 @@
+```kotlin
+private suspend fun handleResponse(response: IdxResponse) {
+    // Check if sign-in is successful.
+    if (response.isLoginSuccessful) {
+        // Exchange the sign-in session token for a connection token.
```

Comment:
```suggestion
        // Exchange the remediation for an OIDC token.
```

Reply:
This isn't consistent wording to what I've seen elsewhere, though it's also not what I've seen. In iOS and in the article I use:

> Exchange the session token for an access token.

Is that correct?

Reply:
We're not exchanging a session token, we're exchanging the issue remediation (containing the `interaction_code`, `grant_type`, `client_id`, `code_verifier`).

Session token is a legacy V1 concept, not OIE.

Reply:
OK, thanks. I've updated that throughout the guide and it will be in the next commit.